auditlog/data/ir_cron.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+
+        <record id="ir_cron_auditlog_autovacuum" model="ir.cron">
+            <field name='name'>Auto-vacuum audit logs</field>
+            <field name='interval_number'>1</field>
+            <field name='interval_type'>days</field>
+            <field name="numbercall">-1</field>
+            <field name="active">True</field>
```

Comment:
Set this to False by default, so that it does not suprise any current users that their logs start to be wiped after half a year?